src/components/mainMenu/DefaultItems.tsx

```diff
@@ -257,29 +255,3 @@ export const Socials = () => (
   </>
 );
 Socials.displayName = "Socials";
-
-export const LiveCollaboration = ({
-  onSelect,
-  isCollaborating,
-}: {
-  onSelect: () => void;
-  isCollaborating: boolean;
-}) => {
-  // FIXME Hack until we tie "t" to lang state
-  // eslint-disable-next-line
-  const appState = useExcalidrawAppState();
-  return (
-    <DropdownMenuItem
-      dataTestId="collab-button"
-      icon={UsersIcon}
-      className={clsx({
-        "active-collab": isCollaborating,
-      })}
-      onSelect={onSelect}
-    >
-      {t("labels.liveCollaboration")}
-    </DropdownMenuItem>
-  );
-};
-
-LiveCollaboration.displayName = "LiveCollaboration";
```

Comment:
this still makes sense to keep exporting?

Reply:
But like I said, we're already exporting half the UI — the top right button — so it only makes sense to export the second one.

And host apps would also loose i18n (at least until we export it. But previous argument still stands.)